pkg/utils/v1alpha1/maya.go

```diff
@@ -211,12 +215,12 @@ func ResizeVolume(
 	if (desiredSize).Cmp(cvcActualSize) == 0 {
```

Comment:
These changes are not related to this PR but it looks like a bug. Here we are returning success by comparing with `cvc.spec.capacity` but we should compare it with `cvc.Status.Capacity` 
cc: @payes

Reply:
I think we are doing a comparison with `cvc.Status.Capacity`

Reply:
Ignore this comment